packages/node_modules/@node-red/editor-client/src/js/ui/editors/json.js

```diff
@@ -498,7 +507,25 @@
                 open: function(tray) {
                     var trayBody = tray.find('.red-ui-tray-body');
                     var dialogForm = RED.editor.buildEditForm(tray.find('.red-ui-tray-body'),'dialog-form',type,'editor');
-
+                    var jsonButtons = options.jsonButtons || [];
+                    if (jsonButtons.length) {
+                        jsonButtons.forEach(function (button) {
+                            var element = $('<button type="button" class="red-ui-button red-ui-button-small"> </button>')
+                                .insertBefore("#node-input-json-reformat")
+                                .on("click", function (evt) {
+                                    evt.preventDefault();
+                                    if (button.click !== undefined) {
+                                        button.click(evt);
```

Comment:
```suggestion
                                        button.click.call(element, evt);
```
Need to call the function with `this` set to the actual button element - so that `$(this)` evaluates to the right thing. Otherwise you get an error in the console when clicking the Copy button.